Fable.Remoting.Client/Extensions.fs

```diff
@@ -14,8 +14,8 @@ module InternalUtilities =
     [<Emit("new Uint8Array($0)")>]
     let createUInt8Array(x: 'a) : byte[]  = jsNative
     /// Creates a Blob from the given input string
-    [<Emit("new Blob([$0.buffer], { type: $1 })")>]
-    let createBlobFromBytesAndMimeType (value: byte[]) (mimeType: string) : Blob = jsNative
+    [<Emit("new Blob([$0], { type: $1 })")>]
+    let createBlobWithMimeType (value: U2<byte[], string>) (mimeType: string) : Blob = jsNative
```

Comment:
`.buffer` wasn't entirely correct, because a typed array could be but a slice of the underlying `ArrayBuffer`, so this would create a `Blob` containing unwanted data.